src/redux/birds/birds.sagas.js

```diff
@@ -166,6 +166,16 @@ export function* goNextLevel() {
         )
           .then((response) => response.json())
           .then((value) => data.push(value));
+        yield fetch(
+          `https://www.flickr.com/services/rest/?method=flickr.photos.search&api_key=7ecd0010d836ddcdfb59aa1364622ed5&tag_mode=all&extras=url_m&format=json&nojsoncallback=1&tags=свиристель`
```

Comment:
don't repeat yourself: 
- use constants for the first part of the URL, 
- add function for yield fetch (the code is repetitive)
- API key uses from env file (you need dotenv library)